src/main/environment/common_ci.properties

```diff
@@ -160,6 +160,9 @@ grievanceDataSyncDuration = @env.GRIEVANCE_DATA_SYNC_DURATION@
 springdoc.api-docs.enabled=false
 springdoc.swagger-ui.enabled=false
 
+
+isProduction=false
```

Comment:
Hey this can't be false.
We should ideally be taking the variable from CI environment.
Use @@

@ravishanigarapu please check.